appcheck/firebase-appcheck-debug-testing/firebase-appcheck-debug-testing.gradle

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+plugins {
+    id 'firebase-library'
+}
+
+firebaseLibrary {
+    testLab.enabled = true
+    publishSources = false
```

Comment:
We want to change this to `true`, but since `true` is the default, please remove this line. here and throughout

Reply:
Done -- chatted offline and `publishSources` defaults to `false` (and `publishJavadoc` defaults to `true`), so changed `publishSources` to `true` for the App Check gradle files.